### ISSUE TITLE
Schema view empty even though json view is not

### ISSUE THREAD

**Issue by wighawag** [enzonotario/vitepress-openapi#110]
### Current behavior

On some route, the schema view is empty even though the json view is not

### Desired behavior

I expect the schema view to always represent the type

### Steps to reproduce

```
git clone https://github.com/wighawag/fuzd.git
cd fuzd
git checkout feat/vitepress-openapi
pnpm i
pnpm docs:dev
# navigate to http://localhost:5173/api/post-api-scheduling-scheduleExecution.html
# click on Schema view for Request Body, see it is empty
```

**Comment by enzonotario** [comment]
Hi @wighawag , I'm working on it in #122 , can you please validate it via [pkg-pr-new release](https://github.com/enzonotario/vitepress-openapi/pull/122#issuecomment-2496549108)?

So far I see it works fine on [StackBlitz](https://stackblitz.com/edit/enzonotario-vitepress-openapi-starter-4bn1kd?file=public%2Fopenapi.json), but it's a pretty big refactor, so I'd appreciate your review and feedback. Thanks!

**Comment by wighawag** [comment]
Cool,

I just check the  stackblitz and I see that the json contains lots of null for the field that should be strings :

Both in Request Body and the Response body

example in `/api/scheduling/scheduleExecution`

```json
{
    "type": "time-locked",
    "chainId": null,
    "slot": null,
    "onBehalf": null,
    "payload": null,
    "timing": {
        "type": "fixed-round",
        "expiry": null,
        "assumedTransaction": {
            "hash": null,
            "nonce": null,
            "broadcastTime": null
        },
        "scheduledRound": null,
        "expectedTime": null
    },
    "paymentReserve": null
}
```

**Comment by enzonotario** [comment]
Good catch! It was returning a `null` example. You can see this fixed in [StackBlitz](https://stackblitz.com/edit/enzonotario-vitepress-openapi-starter-8jhgcw?file=public%2Fopenapi.json).

Thanks!

**Comment by enzonotario** [comment]
Ups, it wasn't supporting `oneOf` as items in an array. It's now fixed in this [StackBlitz](https://stackblitz.com/edit/enzonotario-vitepress-openapi-starter-ednwsr?file=public%2Fopenapi.json).

Currently, it only takes the first `oneOf` element. In the future, we can add a selector or another method to choose from the other `oneOf` options.

Thanks!

**Comment by wighawag** [comment]
Cool, yes the json example having only one option is fine

Found another issue in `/api/execution/remoteAccount/{chainId}/{account}`

the response schema omit the data field which is defined as `{}` in openapi.json

Also, the json example is invalid:

```
{"success": true,"account": {"derivationParameters": {"type": "string",null},"address": "string"}}
```

**Comment by enzonotario** [comment]
Oh nice, it wasn't supporting properties without types. It's now fixed in this [StackBlitz](https://stackblitz.com/edit/enzonotario-vitepress-openapi-starter-hgv2wc?file=public%2Fopenapi.json).

Thanks!

**Comment by wighawag** [comment]
Cool, after looking again at all the endpoint, it looks good :)

**Comment by enzonotario** [comment]
Nice, it's now fixed in [v0.0.3-alpha.49](https://github.com/enzonotario/vitepress-openapi/releases/tag/v0.0.3-alpha.49).

Many thanks!
